client/extensions/woocommerce/state/sites/coupons/actions.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Internal dependencies
+ */
+import {
+	WOOCOMMERCE_COUPONS_REQUEST,
+	WOOCOMMERCE_COUPONS_PAGE_UPDATED,
+} from 'woocommerce/state/action-types';
+
+export function fetchCouponsPage( siteId, pageIndex = 1, perPage = 10 ) {
+	return { type: WOOCOMMERCE_COUPONS_REQUEST, siteId, pageIndex, perPage };
+}
+
+export function couponsPageUpdated( siteId, pageIndex, coupons, totalPages, totalCoupons ) {
+	return { type: WOOCOMMERCE_COUPONS_PAGE_UPDATED, siteId, pageIndex, coupons, totalPages, totalCoupons };
```

Comment:
I think the general pattern we wanted to go with here was `*_REQUEST` and `*_RECEIVE` actions. I had a similar comment over [here](https://github.com/Automattic/wp-calypso/pull/17922#discussion_r137664494).

Reply:
`_RECEIVE` actions aren't so desirable anymore, as we push for further separation between the code which needs data, and the code which does the actual fetching. The preference now is for more meaningful type names. That's why I chose this type name here.